src/makeRobotAccountant.js

```diff
@@ -6,7 +6,27 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  function getSum(x) {
+    return function(y) {
+      count++;
+
+      const sum = x + y;
+
+      if (count <= 3) {
+        return sum;
+      }
+
+      if (count % 2 !== 0) {
+        return sum;
+      }
+
+      return `Bzzz... Error!`;
```

Comment:
also, you'd rather inverse your if-statement
```
if (...) { // if something goes wrong
  // we stop doing what we're doing
}

// otherwise – we do what we need
```
this pattern looks more appropriate